### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by jshorland** [ushahidi/platform#1398]
System should notify you about:
  - [ ] Tasks assigned to you 
System should provide a view where you can see all of your notifications (and alerts?)
- [ ] New notifications are “unread”; they’re marked “read” upon opening the “Notifications” view and displaying “unread” notifications.
Notifications will always appear in web client and mobile apps and can be sent email or SMS if configured
Notifications configuration UI
- [ ] Notifications (and alerts?) config needs its own view in Settings

Questions:
*how do we handle non-authenticated users? 
*will different roles see different things?
*creating an interaction point for all people interacting with the platform is a lot more complicated
*dev estimate: VERY LARGE and needs to be better defined before we can even really estimate just how large this is.

THIS WILL NEED FURTHER SCOPING, DESIGN, AND STAGING FOR USER TESTING

Aha! Link: https://ushahiditeam.aha.io/features/PROD-665

**Comment by rjmackay** [comment]
I'm curious what the justification for "Change “Notifications” to Alerts" is?
Alerts is familiar for v2, but I would have thought Notifications is more familiar from other we apps? (clearly I have no data either way)

**Comment by ClearLakeHABMap** [comment]
Alert seems like an important aspect of the system coming from v2. I am torn between using v2 for this feature alone. Are there any current work around tools that replicate the feature?

**Comment by jshorland** [comment]
To do: we need to split this out into two different issues
1. alerts -- these are communications from the system about new posts that are important to you or urgent
2. notifications -- these are communications from the system about new things that have happened within the deployment (you've been assigned to do something, your post has been edited/published/etc)
